packages/pn-personagiuridica-webapp/src/component/Contacts/SpecialContacts.tsx

```diff
@@ -141,8 +164,8 @@ const SpecialContacts = ({ recipientId, legalAddresses, courtesyAddresses }: Pro
 
   const initialValues = useMemo(
     () => ({
-      sender: '',
-      addressType: addressTypes.find((a) => a.show)?.id as LegalChannelType | CourtesyChannelType,
+      sender: { id: '', name: '' },
```

Comment:
add "as Party" in order to provide correct type for this attribute

Reply:
do it